src/app/services/globals/globals.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from "@angular/core";
+import { cacheSettings } from "@services/cache/cache-settings";
+
+/**
+ * Global functions accessible in the console dev tools
+ */
+@Injectable({ providedIn: "root" })
+export class GlobalsService {
+  private globals = {};
+  // TODO namespace is a bad name
+  private namespace = "__bawWorkbenchClient";
+
+  public initialize(): void {
+    window[this.namespace] = this.globals;
+    this.logToConsole(`
+      ~~~~~~~~~~~~~~~~~~~~
+      BAW Workbench Client
+      ~~~~~~~~~~~~~~~~~~~~
+
+      Some functionality of this website can be toggled from the dev tools. A list of commands and their usages can be seen below:
+      - ${this.toggleCaching()}
+      - ${this.toggleCacheLogging()}
+    `);
+
+    this.toggleCacheLogging();
```

Comment:
toggle functions mutating state just to read the current value is not great

suggestions:



just use getters/setters

and optionally have the toggle functions

have a jquery-esq function.caching() returns value .caching(true|false) sets values

have a .toggle* and a .get* function